docs/_includes/toggle-button-group.html

```diff
@@ -0,0 +1,1349 @@
+
+    <p>The component allows for single or multiple selections based on need. However, when using Skin by itself you will need to have your own <span class="highlight">JavaScript</span> to establish those rules based on the specific needs for each implementation context.</p>
+
+    <p>Effort was taken in these docs to provide a good amount of helpful information. Since this component has an atomic piece (<a href="#toggle-button">Toggle Button</a>), there may be some information crossover. Though efforts were taken to isolate the documentation for this parent component, at various touchpoints between the two components, there may be some redundancy. In other portions, where immediate information appears to be missing here, that information may reside in its child component. Please refer to <a href="#toggle-button">Toggle Button</a> docs in these situations.</p>
+
+    <h3>Accessibility</h3>
+
+    <p>The preferred implementation for accessibility includes an on-screen heading and the list associated with the heading (a). The second option is an off-screen heading with the same association (b). The third is to use the label directly on the list element (c).</p>
+
+    <h4>On-screen Heading with Associated List (a - preferred)</h4>
+    <div class="demo">
+        <div class="demo__inner">
+            <h5 id="toggle-button-group-a">Select Shoe Size</h5>
+            <ul aria-labelledby="toggle-button-group-a" class="toggle-button-group" data-columns="2">
+                <li>
+                    <button type="button" class="toggle-button" aria-pressed="false">
+                        <span class="toggle-button__content">
+                            <span class="toggle-button__title">8</span>
+                        </span>
+                    </button>
+                </li>
+                <li>
+                    <button type="button" class="toggle-button" aria-pressed="false">
+                        <span class="toggle-button__content">
+                            <span class="toggle-button__title">8.5</span>
+                        </span>
+                    </button>
+                </li>
+            </ul>
+        </div>
+    </div>
+
+{% highlight html %}
+<h5 id="toggle-button-group-a">Select Shoe Size</h5>
+<ul aria-labelledby="toggle-button-group-a" class="toggle-button-group" data-columns="2">
+    <li>
+        <button type="button" class="toggle-button" aria-pressed="false">
+            <span class="toggle-button__content">
+                <span class="toggle-button__title">8</span>
+            </span>
+        </button>
+    </li>
+    <li>
+        <button type="button" class="toggle-button" aria-pressed="false">
+            <span class="toggle-button__content">
+                <span class="toggle-button__title">8.5</span>
+            </span>
+        </button>
+    </li>
+</ul>
+{% endhighlight %}
+
+    <h4>Off-screen Heading with Associated List (b)</h4>
```

Comment:
Actually, I would argue that we only need b and do not need c at all. We should provide what the "default" approach should be. If we decide that it should be a clipped element, that is what we should have. 
We have this in other places where we only have 1 way of doing it and it is what we recommend. Its not that the others are wrong, but we have to support and document multiple variations when it can be achieved by only 1.